includes/class-ckwc-order.php

```diff
@@ -457,7 +459,7 @@ public function send_purchase_data( $order_id, $status_old = 'new', $status_new
 			// we honor the integration's "Name Format" setting.
 			'first_name'       => $this->name( $order ),
 			'currency'         => $order->get_currency(),
-			'transaction_time' => $order->get_date_created()->date( 'Y-m-d H:i:s' ),
+			'transaction_time' => $order->get_date_created(),
```

Comment:
The WordPress Libraries accept a `DateTime`object here, not a date string, hence the removal of `->date`.